iroha-lib/README.md

```diff
@@ -1,16 +1,118 @@
+- [x] [addSignatory](https://iroha.readthedocs.io/en/main/develop/api/commands.html#add-signatory)
+- [x] [appendRole](https://iroha.readthedocs.io/en/main/develop/api/commands.html#append-role)
+- [x] [createAccount](https://iroha.readthedocs.io/en/main/develop/api/commands.html#create-account)
+- [x] [createAsset](https://iroha.readthedocs.io/en/main/develop/api/commands.html#create-asset)
+- [x] [createDomain](https://iroha.readthedocs.io/en/main/develop/api/commands.html#create-domain)
+- [x] [createRole](https://iroha.readthedocs.io/en/main/develop/api/commands.html#create-role)
+- [x] [detachRole](https://iroha.readthedocs.io/en/main/develop/api/commands.html#detach-role)
+- [x] [grantPermission](https://iroha.readthedocs.io/en/main/develop/api/commands.html#grant-permission)
+- [x] [removeSignatory](https://iroha.readthedocs.io/en/main/develop/api/commands.html#remove-signatory)
+- [x] [revokePermission](https://iroha.readthedocs.io/en/main/develop/api/commands.html#revoke-permission)
+- [x] [setAccountDetail](https://iroha.readthedocs.io/en/main/develop/api/commands.html#set-account-detail)
+- [x] [setAccountQuorum](https://iroha.readthedocs.io/en/main/develop/api/commands.html#set-account-quorum)
+- [x] [subtractAssetQuantity](https://iroha.readthedocs.io/en/main/develop/api/commands.html#subtract-asset-quantity)
+- [x] [transferAsset](https://iroha.readthedocs.io/en/main/develop/api/commands.html#transfer-asset)
+- [x] [сompareAndSetAccountDetail](https://iroha.readthedocs.io/en/main/develop/api/commands.html#compare-and-set-account-detail)
+- [x] [removePeer](https://iroha.readthedocs.io/en/main/develop/api/commands.html#remove-peer)
+
+
+## Queries
+
+- [x] [getAccount](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-account)
+- [x] [getAccountAssetTransactions](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-account-asset-transactions)queries.html#get-account-assets)
+- [x] [getAccountDetail](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-account-detail)
+- [x] [getAccountTransactions](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-account-transactions)
+- [x] [getTransactions](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-transactions)
+- [x] [getSignatories](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-signatories)
+- [x] [getAssetInfo](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-asset-info)
+- [x] [getRoles](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-roles)
+- [x] [getRolePermissions](https://iroha.readthedocs.io/en/main/develop/api/queries.html#get-role-permissions)
+
+
+## Compatibility and release policy
+
+- **master** branch of `hyperledger/iroha-java` is compatible with **master** branch of `hyperledger/iroha`
+- branches `develop*` are compatible with tagged releases of Iroha. Example: `v2.0.0-pre-rc.9`.
+
+
+## Info
+- Iroha API might and will change.
```

Comment:
I think that this can be removed - Iroha 1 rather will not have api changes soon (as I know). Probably next year internship project will introduce small changes, but for changes once a year it is better not to "scary" library's users:D.
```suggestion
```

Reply:
Removed. Not necessary.